src/lib/logical_query_plan/delete_node.cpp

```diff
@@ -22,7 +22,8 @@ std::string DeleteNode::description() const {
 bool DeleteNode::is_column_nullable(const ColumnID column_id) const { Fail("Delete does not output any columns"); }
 
 const std::vector<std::shared_ptr<AbstractExpression>>& DeleteNode::column_expressions() const {
-  return _dummy_column_expressions;
+  static std::vector<std::shared_ptr<AbstractExpression>> dummy_column_expressions;
+  return dummy_column_expressions;
```

Comment:
When I read "dummy" here, I thought it had something to do with the dummy node. Now that it is in the same method, it's not that much of a problem, but in my opinion `empty_vector` would express better that we would like to `return {}` but can't. Won't mind if you disagree and simply merge though.